src/collection.js

```diff
@@ -554,9 +555,20 @@ export default class Collection {
    *
    * @return {Object}
    */
-  gatherLocalChanges() {
+  gatherLocalChanges(options={forceAll: false}) {
+    var forceSync = Promise.resolve();
+    if (options.forceAll) {
+      forceSync = this.list({filter: {_status: "synced"}})
+        .then(synced => {
+          synced.map(r => {
+            r._status = "created";
+            delete r.lastModified;
+          });
```

Comment:
I'm also confused with a `map` operation not being returned by the function. Please explain :)